src/styles/annotator/annotator.scss

```diff
@@ -56,6 +56,10 @@ var.$base-font-size: 14px;
     position: relative;
   }
 
+  .is-closed {
```

Comment:
Hmm, rather than adding a new CSS class and logic to add/remove it, perhaps it would be easier in this scenario to leverage the existing `.annotator-collapsed` class that is set on the parent `.annotator-frame` with a descendant selector?